Classes/Service/ImageResizer.php

```diff
@@ -310,7 +310,41 @@ public function processFile(
         if ((bool)$ruleset['auto_orient'] === true) {
             $orientation = ImageUtility::getOrientation($fileName);
             $isRotated = ImageUtility::isRotated($orientation);
-            $gifCreator->scalecmd = '-auto-orient ' . $gifCreator->scalecmd;
+
+            if ((bool)$ruleset['auto_orient_custom'] === true) {
+                switch ((int) $orientation) {
+
```

Comment:
Please drop all those useless line breaks between cases, they do not bring anything and are not used either in the rest of the codebase.